theorems/T000275.md

```diff
@@ -0,0 +1,12 @@
+---
+uid: T000275
+if:
+  and:
+  - P000133: true
+  - P000036: true
+  - P000026: true
+then:
+  P000027: true
+---
+
+Let $X$ be a connected linearly ordered topological space with countable dense subset $S$. Then the collection of all open intervals and rays of the form $(a, b)$ with $a,b\in S\cup \{-\infty, \infty\}$ forms a countable base for $X$.
```

Comment:
We should add how the connected hypothesis is used in the argument.  Also need some reference.

Reply:
See [72d5867](https://github.com/pi-base/data/pull/147/commits/72d586700b2e467cff948c839f29d0047e8004aa) for my addition on connecteness.